includes/Import.php

```diff
@@ -819,6 +814,9 @@ private function processTitle( $text ) {
 			# Do not import if the importing wiki user cannot create this page
 			$this->notice( 'import-error-create', $title->getPrefixedText() );
 			return false;
+		} elseif( $title->getNamespace() == NS_MEDAWIKI  ) {
+			# Do not import if pages in the MediaWikia Namespace
+			return false;
```

Comment:
This prevents importing pages from the MW namespace which was the intent MAIN-5118 which was reverted.